docs/modules/reference/pages/service-assurance/monitors/DeviceConfiMonitor.adoc

```diff
@@ -0,0 +1,57 @@
+
+= DeviceConfigMonitor
```

Comment:
```suggestion
= DeviceConfigMonitor
```
Is this correct name? Could we rename .adoc file and the nav file entry  to include the "g"?